src/content/developers/docs/index.md

```diff
@@ -42,5 +42,6 @@ If this is your first attempt at Ethereum development, we recommend starting at
 ### Advanced {#advanced}
 
 - [Token standards](/developers/docs/standards/tokens/)
+- [Miner extractable value](/developers/docs/miner-extractable-value/)
```

Comment:
Perhaps we should just go with /mev for this page URI given the movement to "maximal extractable value" given this isn't just a miner/POW issue. Gives it a bit more flexibility / longevity.

Reply:
Agreed, changed

Reply:
agreed, good call!